README.rst

```diff
@@ -1,151 +1,176 @@
+publishing the changes in the course.
 
-2. Run i18n tools to create Raw message catalogs
-=================================================
 
-This cookiecutter template offers multiple make targets which are shortcuts to
-use `edx-i18n-tools <https://github.com/openedx/i18n-tools>`_.
+View from the Learning Management System (LMS)
+**********************************************
 
-After marking strings as translatable we have to create the raw message catalogs.
-These catalogs are created in ``.po`` files. For more information see
-`GNU PO file documentation <https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html>`_.
-These catalogs can be created by running::
+The students can view and download the files and folders that have been added to the
+component from the LMS. The download can be individual or multiple (as a zip file).
 
+.. image:: https://github.com/eduNEXT/xblock-filesmanager/assets/64033729/fb174233-0b5e-4506-8335-e7197121f2f6
+   :alt: View from the LMS
 
-    $ make extract_translations
 
-The previous command will create the necessary ``.po`` files under
-``xblock-filesmanager/filesmanager/locale/en/LC_MESSAGES/text.po``.
-The ``text.po`` file is created from the ``django-partial.po`` file created by
-``django-admin makemessages`` (`makemessages documentation <https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#message-files>`_),
-this is why you will not see a ``django-partial.po`` file.
+Experimenting with this XBlock in the Workbench
+************************************************
 
-3. Create language specific translations
-==============================================
+`XBlock`_ is the Open edX component architecture for building custom learning
+interactive components.
 
-3.1 Add translated strings
----------------------------
+.. _XBlock: https://openedx.org/r/xblock
 
-After creating the raw message catalogs, all translations should be filled out by the translator.
-One or more translators must edit the entries created in the message catalog, i.e. the ``.po`` file(s).
-The format of each entry is as follows::
+You can see the Images Gallery component in action in the XBlock Workbench.
+Running the Workbench requires having docker running.
 
-    #  translator-comments
-    A. extracted-comments
-    #: reference…
-    #, flag…
-    #| msgid previous-untranslated-string
-    msgid 'untranslated message'
-    msgstr 'mensaje traducido (translated message)'
+.. code:: bash
 
-For more information see
-`GNU PO file documentation <https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html>`_.
+    git clone git@github.com:eduNEXT/xblock-filesmanager
+    virtualenv venv/
+    source venv/bin/activate
+    cd xblock-filesmanager
+    make upgrade
+    make install
+    make dev.run
 
-To use translations from transifex use the follow Make target to pull translations::
+Once the process is done, you can interact with the Images Gallery XBlock in
```

Comment:
```suggestion
Once the process is done, you can interact with the Files Manager XBlock in
```
You can also fix the rest of the images gallery references

Reply:
Done